src/components/Table/Table.stories.tsx

```diff
@@ -206,6 +221,16 @@ const Template: ComponentStory<typeof Table> = (args) => {
               key={row.applicationNr}
               rowData={{ applicationNr: row.applicationNr }}
             >
+              {args.selectRows && (
+                <TableCell radiobutton={true}>
+                  <RadioButton
+                    name={row.applicationNr}
+                    onChange={(event) => handleRadioButton(event)}
+                    value={row.applicationNr}
+                    checked={checkSelectedValue(row)}
+                  ></RadioButton>
```

Comment:
Foreslår å legge til attributtene `label` og `hideLabel` her. Med `hideLabel` satt, vil teksten i `label` bli gjort tilgjengelig for skjermlesere selv om den ikke vises på skjermbildet.